docker-compose.yml

```diff
@@ -223,6 +223,11 @@ services:
     restart: on-failure:2
     mem_limit: 1g
     working_dir: /mnt/Dev
+    environment:
+      PHP_EXTENSION_XDEBUG: 1
+      PHP_INI_XDEBUG__REMOTE_PORT: 10000
+      PHP_IDE_CONFIG: "serverName=plumbus.com"
```

Comment:
```suggestion
      PHP_IDE_CONFIG: "serverName=plumbus.com"
```
Why plumbus.com ?

Reply:
I think this line is not required at all